lte/gateway/c/core/oai/tasks/amf/Registration.cpp

```diff
@@ -727,15 +727,13 @@ static int registration_accept_t3550_handler(zloop_t* loop, int timer_id,
           "T3550: timer has expired retransmitting registration accept\n");
       amf_send_registration_accept(amf_ctx);
     } else {
-      /* Abort the registration procedure */
-      OAILOG_ERROR(
-          LOG_AMF_APP,
-          "T3550: Maximum retires:%d, for registration accept done hence Abort "
-          "the registration "
-          "procedure\n",
-          registration_proc->retransmission_count);
-      // To abort the registration procedure
-      amf_proc_registration_abort(amf_ctx, ue_amf_context);
+      /* 5.5.1.2.8 abnormal case on network side */
```

Comment:
Is this the case where a new 5G-GUTI is not included in  REGISTRATION ACCEPT message?

Reply:
@pruthvihebbani, thanks for the review comments.
This is not the case where 5G GUTI is not included. Here 5g GUTI was included in the REGISTRATION ACCEPT message, but somehow UE was not responding to the REGISTRATION ACCEPT.

Reply:
Hi @pruthvihebbani , we followed ETSI TS 124 501 V16.5.1 - sec 5.5.1.2.8,

![24 501 5 5 1 2 8](https://user-images.githubusercontent.com/95931489/174306802-5fa7afec-6ed1-4766-861b-17a81bbe5ed3.PNG)
If we don't move UE to REGISTERED state, UE is not able to register and continuously triggered service request failing at AMF. Ultimately UE couldn't be able to attach the core. 
Please find below pcap snap:
![4-reg_accept_pcap_snap](https://user-images.githubusercontent.com/95931489/174307774-635bccaa-3cd4-4225-aa1e-1c682811c30b.PNG)

Reply:
Sounds good. I was referring to TS 124 501 - V15.1.0 where the approach is different. Can you please add the spec and version number under comments

Reply:
comment addressed. We have updated summary with the reference specs version and section details.

Reply:
I meant add the comment in the file at line 730 - // ETSI TS 124 501 V16.5.1 - sec 5.5.1.2.8 abnormal case on network side

Reply:
addressed the comment, added specs details in the file.